### DISCUSSION TITLE
How can i set "news" on strapi collection type

### DISCUSSION THREAD

**Discussion by chrisk8er** [strapi/strapi#7621] [Legacy Topics]
Hi there, i just trying to create some collection type. but strapi can not allow me to add "news" name on it.

**Comment by derrickmehaffy** [comment]
Yup there is currently an issue about certain names that have the same "name" for both their singular and plural forms. `news` in this case is both singular and plural. For the moment I would recommend you use `news item` which will be `news items` as a plural form.

See this issue: https://github.com/strapi/strapi/issues/1198
